app/domain/finders/content.rb

```diff
@@ -22,42 +22,75 @@ def call
   ALL = 'all'.freeze
   NONE = 'none'.freeze
 
-  attr_reader :organisation_id, :document_type, :date_range, :search_term
+  attr_reader :filter
```

Comment:
Is this attribute access from outside this class? if not, can we remove the attr_reader and use the `@filter` instance variable, so we are consistent with date_range, page, etc...

Reply:
It is a `private` `attr_reader`. It feels like a good practice to me. It is not accessible from the outside, and we don't overuse instance variables within the class.